apps/examples/testcase/le_tc/kernel/tc_clock.c

```diff
@@ -237,6 +244,24 @@ static void tc_clock_clock_abstime2ticks(void)
 	TC_SUCCESS_RESULT();
 }
 
+#ifdef CONFIG_RTC
+/**
+* @fn                   :tc_clock_clock_synchronize
+* @brief                :Synchronize the system timer to a hardware RTC
+* @scenario             :First set time to 1 day ahead and then synchronie it back to real time
+* API's covered         :clock_synchronize
+* Preconditions         :none
+* Postconditions        :none
+* @return               :void
+ */
+static void tc_clock_clock_synchronize(void)
+{
+
+	clock_synchronize();
```

Comment:
You'd better check that the initial value setting is correct or not.

Reply:
It is not a testcase. If you want to add this function, let's do like below.
1. check current time.
2. set a clock to different time.
3. call ```clock_synchronize```
4. check new time.

Reply:
because clock_synchronize()  - Synchronizes the system timer to a hardware RTC
& clock_settime() also sets RTC time
we cannot check it like this as it will be synchronized to the new set time.

Reply:
I am sorry I don't understand.
If we call this, g_basetime will be changed. And same for clock_settime.
Anyway, if we don't have any way to check this, let's remove this tc function.